ludwig/features/image_feature.py

```diff
@@ -16,32 +16,35 @@
 # ==============================================================================
 import logging
 import os
-import sys
 from functools import partial
 from multiprocessing import Pool
 from typing import Union
 
 import numpy as np
+import torch
+import torchvision
 # import tensorflow as tf
 
 from ludwig.constants import *
 from ludwig.encoders.image_encoders import ENCODER_REGISTRY
 from ludwig.features.base_feature import InputFeature
 from ludwig.utils.data_utils import get_abs_path
 from ludwig.utils.fs_utils import upload_h5
-from ludwig.utils.image_utils import greyscale
-from ludwig.utils.image_utils import num_channels_in_image
-from ludwig.utils.image_utils import resize_image
-from ludwig.utils.image_utils import get_image_from_path, read_image
+from ludwig.utils.image_utils import greyscale, num_channels_in_image,\
+    resize_image, get_image_from_path, read_image
 from ludwig.utils.misc_utils import set_default_value
 
 logger = logging.getLogger(__name__)
 
-
+# TODO(shreya): Confirm if it's ok to do per channel normalization
+# TODO(shreya): Also confirm if this is being used anywhere
+# TODO(shreya): Confirm if ok to use imagenet means and std devs
 image_scaling_registry = {
     'pixel_normalization': lambda x: x * 1.0 / 255,
-    'pixel_standardization': lambda x: tf.map_fn(
-        lambda f: tf.image.per_image_standardization(f), x)
+    'pixel_standardization': partial(
+        torchvision.transforms.functional.normalize,
+        mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]
```

Comment:
Is this equivalent to what `tf.image` was doing?

Reply:
Not really -- I couldn't find the equivalent of this in torch. I've standardized images using the ImageNet means and std devs right now, which is usually a reasonable standardization.

Another alternative -- I could change this to implement my own per image standardization. Lmk what you think!